batch-processing/src/test/java/org/wildfly/quickstarts/batch-processing/BasicRuntimeIT.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2023 JBoss by Red Hat.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.quickstarts.helloworld;
```

Comment:
can we use a package name specific for this quickstart, like `org.wildfly.quickstarts.batch `

Reply:
that's a bug, thanks @chengfang